README.md

```diff
@@ -1,10 +1,11 @@
-This repository contains the Sphinx source for the SciPy website
-(http://www.scipy.org/).
+# Scipy.org Website Source Code
+ 
+This repository contains the Sphinx source for the [SciPy website](http://www.scipy.org/).
 
 After cloning this repository, run
 
-    $ git submodule init
-    $ git submodule update
+`$ git submodule init`  
+`$ git submodule update`
```

Comment:
Maybe make these into code blocks with three backticks (remove spacing added between for rendering):

``` 
` ` `
$git submodule init
$git submodule update
` ` `
```

Reply:
Sure. Done with it.